src/rendering/RenderUtil.cc

```diff
@@ -727,10 +751,29 @@ void RenderUtilPrivate::CreateRenderingEntities(
           // todo(anyone) make visual updates more generic without using extra
           // variables like entityTemp just for storing one specific visual
           // param?
```

Comment:
@iche033 this comment was left here before I started updating making updates to the thermal camera for a heat signature. `entityTemp`'s usage is now a little more complicated than before, so I believe that this comment doesn't apply anymore. If you agree with me on this, can I go ahead and remove this comment?

Reply:
ok, we can remove this for now. Ideally in the future we have some sort of data struct to store these thermal camera related variables.

Reply:
removed in https://github.com/ignitionrobotics/ign-gazebo/pull/498/commits/a60e5d37ecc39090ce8141472f82b693c8466751